docs/source/peer_ledger_snapshot.md

```diff
@@ -88,7 +88,7 @@ You will see a response similar to:
 Successfully got pending snapshot requests [1000]
 ```
 
-When a snapshot has been generated for a particular block height, the pending request for that block height will no longer appear in the list. You can also verify that a snapshot has been created successfully by looking at the peer logs or by checking the expected location of the snapshot, `{snapshotsFolder}/completed/{channelName}/{lastBlockNumberInSnapshot}` under the directory defined by the `peer.fileSystemPath` in the peer's `core.yaml`.
+When a snapshot has been generated for a particular block height, the pending request for that block height will no longer appear in the list. You can also verify that a snapshot has been created successfully by looking at the peer logs or by checking the expected location of the snapshot. The snapshot root directory is based on the `core.yaml` `ledger.snapshots.rootDir` property. Completed snapshots will be written to a subdirectory based on the channel name and block number of the snapshot: `{ledger.snapshots.rootDir}/completed/{channelName}/{lastBlockNumberInSnapshot}`.
```

Comment:
We can add one more sentence - if the property `ledger.snapshots.rootDir` is not specified in the `core.yaml`, then the default value used is `{peer.fileSystemPath}/snapshots`.

Reply:
done